doc/source/development.rst

```diff
@@ -99,6 +99,47 @@ directory will take effect without reinstalling the package.
 
 .. warning:: if you run ``python setup.py install``, files will be copied from the Ray directory to a directory of Python packages (``/lib/python3.6/site-packages/ray``). This means that changes you make to files in the Ray directory will not have any effect.
 
+Building Ray on Windows (full)
+------------------------------
+
+**Requirements**
+
+- Microsoft Visual Studio 2019 (or Microsoft Build Tools 2019)
+- JDK 15
+- Miniconda 3
+- git for Windows, version 2.31.1 or later
+
+**Steps**
+
+1. Enable Developer mode on Windows 10 systems.
+2. Enable symlinks on git:
+
+.. code-block:: shell
+
+  git config core.symlinks=true
```

Comment:
It might be better to omit this here and instead clone Ray below with
```
git clone -c core.symlinks=true https://github.com/ray-project/ray.git
```
In that case the user doesn't need to worry about whether this config is global or if they use the right command prompt etc.